tests/facility/settings/specimen-definitions/specimen-definitions.spec.ts

```diff
@@ -0,0 +1,275 @@
+import { faker } from "@faker-js/faker";
+import { expect, test } from "@playwright/test";
+import { getFacilityId } from "tests/support/facilityId";
+
+// Use the authenticated state
+test.use({ storageState: "tests/.auth/user.json" });
+
+const SLUG_LENGTH = 5;
+
+test.describe("Specimen Definitions Management", () => {
+  let facilityId: string;
+
+  // Faker data
+  let definitionTitle: string;
+  let definitionDescription: string;
+  let definitionSlug: string;
+
+  // Common navigation before each test
+  test.beforeEach(async ({ page }) => {
+    // Get facility ID for each test run
+    facilityId = getFacilityId();
+
+    // Generate fresh faker values for each test
+    definitionTitle = faker.science.chemicalElement().name;
+    definitionDescription = faker.lorem.sentence();
+    const randomHex = faker.string
+      .hexadecimal({ length: SLUG_LENGTH, prefix: "" })
+      .toLowerCase();
+    definitionSlug = `${faker.science.chemicalElement().symbol.toLowerCase()}-${randomHex}`;
+
+    // Navigate to specimen definitions list
+    const targetUrl = `/facility/${facilityId}/settings/specimen_definitions`;
+    await page.goto(targetUrl);
+  });
+
+  test("should create specimen definition with all required fields", async ({
+    page,
+  }) => {
+    // Click Add Definition button
+    await page.getByRole("button", { name: "Add Definition" }).click();
+
+    // Fill required fields - Title
+    await page.getByRole("textbox", { name: "Title *" }).fill(definitionTitle);
+
+    // Fill required fields - Slug (must be max 25 characters)
+    await page.getByRole("textbox", { name: "Slug *" }).fill(definitionSlug);
+
+    // Fill required fields - Description
+    await page
+      .getByRole("textbox", { name: "Description *" })
+      .fill(definitionDescription);
+
+    // Select status (required field)
+    await page.getByRole("combobox", { name: "Status *" }).click();
+    await page.getByRole("option", { name: "active" }).click();
+
+    // Select Type Collected (required field)
+    await page.getByRole("combobox", { name: "Type Collected *" }).click();
+    await page.getByRole("option").first().click();
+
+    // Submit form
+    await page.getByRole("button", { name: /save/i }).click();
+
+    // Verify it appears in the list with correct data
+    await page
+      .getByRole("textbox", { name: "Search definitions" })
+      .fill(definitionTitle);
+
+    const tableBody = page.locator('[data-slot="table-body"]');
+    await expect(tableBody).toContainText(definitionTitle);
+    await expect(tableBody).toContainText(definitionDescription);
+    await expect(tableBody).toContainText(/active/i);
```

Comment:
- verify the data in the view details page
- verify the data in the edit specimen page